commands/client-setname.md

```diff
@@ -2,14 +2,14 @@ The `CLIENT SETNAME` command assigns a name to the current connection.
 
 The assigned name is displayed in the output of `CLIENT LIST` so that it is possible to identify the client that performed a given connection.
 
-For instance when Redis is used in order to implement a queue, producers and consumers of messages may want to set the name of the connection according to their role.
+For instance when Valkey is used in order to implement a queue, producers and consumers of messages may want to set the name of the connection according to their role.
 
-There is no limit to the length of the name that can be assigned if not the usual limits of the Redis string type (512 MB). However it is not possible to use spaces in the connection name as this would violate the format of the `CLIENT LIST` reply.
+There is no limit to the length of the name that can be assigned if not the usual limits of the Valkey string type (512 MB). However it is not possible to use spaces in the connection name as this would violate the format of the `CLIENT LIST` reply.
```

Comment:
```suggestion
There is no limit to the length of the name that can be assigned if not the usual limits of the string type (512 MB). However it is not possible to use spaces in the connection name as this would violate the format of the `CLIENT LIST` reply.
```